ci-operator/config/openshift/operator-framework-tooling/openshift-operator-framework-tooling-main.yaml

```diff
@@ -0,0 +1,63 @@
+base_images:
+  commitchecker:
+    name: commitchecker
+    namespace: ci
+    tag: "4.15"
+  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.15
+  os:
+    name: centos
+    namespace: origin
+    tag: stream8
+binary_build_commands: GO111MODULE=on GOPROXY=https://proxy.golang.org make build
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: v0.Dockerfile
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/operator-framework-tooling/v0
+    ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15
+  to: olm-v0-bumper
```

Comment:
Noting that https://github.com/openshift/release/blob/3bf0b3ae011debaefefb564ad6f233c380d033f7/ci-operator/jobs/infra-periodics.yaml#L965 references `olm-bumper`. Does this matter?

Reply:
We're going to have both v0 and v1 bumpers. I'll update the job later.